cli/lib/tasks/verify.ts

```diff
@@ -104,18 +113,18 @@ const runSmokeTest = (binaryDir, options) => {
     const stdioOptions = _.extend({}, {
       env: {
         ...process.env,
-        FORCE_COLOR: 0,
+        FORCE_COLOR: '0',
```

Comment:
@AtofStryker I've confused myself on whether this is actually testing what it should be testing. https://force-color.org/ This was the original PR: https://github.com/cypress-io/cypress/pull/28994 

>When this variable is present and not an empty string (regardless of its value), it should force the addition of ANSI color.

Wasn't the original intention to have this evaluate to OFF?

Reply:
We want to turn it off because the colored ANSI output was causing out verification task to fail. This should still be testing what we want

Reply:
It would still evaluate to false (or at least should) because anything piped into `execa` would evaluate to a string as its an env variable in the process. It's the equivalent of doing something like `FORCE_COLOR=0 node index.js` where `process.env.FORCE_COLOR` inside `index.js` would actually be `'0'`

<img width="980" height="400" alt="Screenshot 2025-08-22 at 4 33 28 PM" src="https://github.com/user-attachments/assets/53e6dd1c-ca96-4913-a315-5edd22107ace" />